src/Project/Project.router.ts

```diff
@@ -79,6 +79,15 @@ export class ProjectRouter extends Router {
       server.handleRequest(this.deleteProject)
     )
 
+    /**
+     * Update all projects with coords to null
+     */
+    this.router.post(
```

Comment:
```suggestion
    this.router.delete(
```

May this be a `delete` instead? Post implies creation of resources

Reply:
Delete usually implies that the row is deleted of marked as deleted, in this case the only thing we do is unlink the project to a land, but the project is still accessible by the builder and in a future, by the builder in world, so I see it as an "update". 

However, if you want I can set it to delete

Reply:
If that's the case, I'd prefer to use the delete. It is more aligned with the concept of entities of the REST apis. you are deleting the `coord` property of the `project` entity

Reply:
Would it make sense to just use the already existing `PUT /project/:id` ? I assume the `id` is not available right?

I think I prefer either PUT/DELETE (also because this method in particular is Idempotent)

Reply:
Yep, we don't have the `id ` of all of your projects. Changed to Delete!